src/app/pipes/alm-multiline-truncate.pipe.ts

```diff
@@ -0,0 +1,35 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/*
+ * Usage:
+ *   {{ aParagraph | almMultilineTruncate:wordCount }}
+ *
+ * Example:
+ *  paragraph = "Lorem Ipsum is simply dummy text of the printing and
+ *               typesetting industry. Lorem Ipsum has been the industry's
+ *               standard dummy text ever since the 1500s, when an unknown
+ *               printer took a galley of type and scrambled it to make a
+ *               type specimen book. "
+ *
+ * {{ paragraph | almMultilineTruncate:20 }}
+ *
+ * Output:
+ * "Lorem Ipsum is simply dummy text of the printing and typesetting
+ *  industry. Lorem Ipsum has been the industry's standard dummy..."
+*/
+
+@Pipe({ name: 'almMultilineTruncate' })
+export class AlmMultilineTruncate implements PipeTransform {
+  transform(val: string, wordCount: number): any {
+    if (!val) {
+      return '';
+    }
+    let words = val.split(' ');
```

Comment:
Sometimes words are bigger, and 20 words might not fit in. Why don't we do it with width instead of word count with CSS?